bidon_api/app/services/kafka_event.rb

```diff
@@ -32,6 +34,12 @@ def fill_geo_data!
     params['geo']['country_id'] = geo_data[:country_id]
   end
 
+  def alias_bid_to_show!
```

Comment:
Can you please add deprecation comment, that we should remove support for `show` in the future, after 1.0 for examplle